src/dev/impl/DevToys/ViewModels/Tools/Text/StringUtilities/StringUtilitiesToolViewModel.cs

```diff
@@ -626,6 +634,182 @@ private void ExecuteInverseCaseCommand()
 
         #endregion
 
+        #region AlphabetizeCommand
+
+        public IRelayCommand AlphabetizeCommand { get; }
+
+        private void ExecuteAlphabetizeCommand()
+        {
+            string? text = _textBackup;
+            if (text is null)
+            {
+                return;
+            }
+
+            IOrderedEnumerable<string> lines = text.Split('\r').OrderBy(line => line, StringComparer.CurrentCulture);
+            text = string.Join('\r', lines);
+
+            lock (_lockObject)
```

Comment:
We do this block of code at many places, including in the older code I wrote originally (oops). Could we create a helper method `SetTextWithoutBackup` that ultimately runs this block?

Reply:
You got it! 5f7d9b2ce6596e2680a6fe9a37dc9af09405ed09